main.go

```diff
@@ -67,13 +234,11 @@ func main() {
 
 	if opt.Help {
 		usage(os.Stdout)
-		return
```

Comment:
You probably don't want to remove this return, since it serves to actually exit the program.  Otherwise, `gotcha -h` will print the help and then kvetch about missing arguments.